test/parallel/test-readline-interface-tty-readstream.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+// Refs: https://github.com/nodejs/node/issues/5574
+
+/**
+ * Test that characters from stdin are not randomly lost when using a
+ * `readline.Interface` and a `tty.ReadStream` on `process.stdin`
+ */
```

Comment:
Nit: Please use the `//` style for these kinds of comments, just for consistency